course/templates/course/index.html

```diff
@@ -8,56 +8,56 @@
-            {% include "course/course_cards.html" with instances=my_instances %}
-        </div>
-    </div>
+	<div class="frontpage-panel">
+		<h2 class="panel-heading">{% trans "My courses"%}</h2>
+		<div class="row frontpage">
+			{% include "course/_course_cards.html" with instances=my_instances %}
+		</div>
+	</div>
 {% endif %}
 
 {% if all_instances %}
-    <div class="frontpage-panel">
-        <h2 class="panel-heading">{% trans "All ongoing courses"%}</h2>
-        <div class="row frontpage">
-            {% site_advert as advert %}
-            {% if advert %}
-                <div class="col-sm-6 col-md-4 col-lg-3">
-                    <a href="{{ advert.href }}" target="_blank">
-                        <div class="card advert-card">
-                            <div class="advert-image" aria-hidden="true">
-                                {% if advert.image %}
-                                    <img class="card-img-top" src="{{ advert.image }}" alt="" />
-                                {% else %}
-                                    <div class="card-img-top">
-                                        <i class="glyphicon glyphicon-comment"></i>
-                                    </div>
-                                {% endif %}
-                            </div>
-                            <div class="card-body">
-                                <h3 class="card-title">{{ advert.title }}</h3>
-                                <p class="card-text">{{ advert.text|safe }}</p>
-                            </div>
-                        </div>
-                    </a>
-                </div>
-            {% endif %}
-            {% include "course/course_cards.html" with instances=all_instances %}
-        </div>
-    </div>
+	<div class="frontpage-panel">
+		<h2 class="panel-heading">{% trans "All ongoing courses"%}</h2>
+		<div class="row frontpage">
+			{% site_advert as advert %}
+			{% if advert %}
+				<div class="col-sm-6 col-md-4 col-lg-3">
+					<a href="{{ advert.href }}" target="_blank">
+						<div class="card advert-card">
+							<div class="advert-image" aria-hidden="true">
+								{% if advert.image %}
+									<img class="card-img-top" src="{{ advert.image }}" alt="" />
+								{% else %}
+									<div class="card-img-top">
+										<i class="glyphicon glyphicon-comment"></i>
+									</div>
+								{% endif %}
+							</div>
+							<div class="card-body">
+								<h3 class="card-title">{{ advert.title }}</h3>
+								<p class="card-text">{{ advert.text|safe }}</p>
+							</div>
+						</div>
+					</a>
+				</div>
+			{% endif %}
+			{% include "course/_course_cards.html" with instances=all_instances %}
+		</div>
+	</div>
 {% endif %}
 
 
 <p class="archive-link">
-    {% url 'archive' as url %}
-    {% blocktrans with url=url %}
-    Old courses can be found at the <a href="{{ url }}">course archive</a>.
-    {% endblocktrans %}
+	{% url 'archive' as url %}
+	{% blocktrans with url=url %}
```

Comment:
I'm pretty sure this `blocktrans` was written originally before Django 1.7, as are many others too. Since then, there has been posibility to add `trimmed` argument, which makes the translation step a bit nicer, as it removes extra whitespace. It's pretty neat, I recommend to test it out :) 

```
{% blocktrans trimmed with url=url %}
```

Reply:
Oh awesome, I have been wondering whether there is something like this. That part of the block-translation keeps bugging me every time. Will use this from now on!